### DISCUSSION TITLE
Release Plans for Quarkus 3.7, 3.8 and 3.9

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#37834] [Quarkus Blog/Website]
# Release Plans for Quarkus 3.7, 3.8 and 3.9

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/our-plans-for-quarkus-3-7-3-8-3-9-released/

**Comment by adampoplawski** [comment]
Hello
Is there a place where I can find if there are plans to migrate to hibernate 6.4 for orm and reactive in 3.7?

**Reply by maxandersen** [reply]
Hibernate 6.4 is already used in main.

**Reply by Postremus** [reply]
More specificly, 3.7 contains the ORM 6.4 Upgrade. https://github.com/quarkusio/quarkus/pull/36978

**Reply by adampoplawski** [reply]
Thank you!

**Comment by JanWiemer** [comment]
Is there a general plan about the cadence of LTS releases?

**Reply by yrodiere** [reply]
See https://quarkus.io/blog/lts-releases/

> LTS versions will be supported for a period of 12 months

> We aim to release a new LTS version every 6 months.
